frontend/constants/networks.ts

```diff
@@ -17,6 +18,7 @@ const NETWORKS = {
     explorerURL: 'https://explorer.execution.mainnet.lukso.network',
     universalGraveForwarder: '0x433908ce6457b302a6452257Bc40e466d95c78E9',
     lsp1UrdVault: '0x9292dAf1cdc3d03a1A0BbD4B3319C49A3B91d703',
+    baseUrl: 'https://universalgrave.com',
```

Comment:
anyway we can do it so we can use it on preview?

Reply:
Maybe if we change it to query param in the future that way we can also test both without restarting